### PR TITLE
Bpk 3774 add holidays view

### PR DESCRIPTION
**Pull request by tiagoengel**
# Footer view

Screenshot_1585235990
Screenshot_1585235980

# RTL

Screenshot_1585236199

+ [ ] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/master/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] `UNRELEASED.md`
+ [x] `README.md`
+ [x] Tests
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: app/src/androidTest/java/net/skyscanner/backpack/calendar/BpkCalendarTest.kt [hunk 1]
```diff
@@ -78,7 +77,6 @@ class BpkCalendarTest : BpkSnapshotTest() {
   }
 
   @Test
-  @FlakyTest
```
Comment (tiagoengel) [inline app/src/androidTest/java/net/skyscanner/backpack/calendar/BpkCalendarTest.kt]: This was left over from before we started to use firebase, they are not flaky anymore.

File: Backpack/src/main/java/net/skyscanner/backpack/calendar/view/HighlightedDaysMonthFooter.kt [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+import android.widget.LinearLayout
+import androidx.annotation.ColorInt
+import androidx.constraintlayout.widget.ConstraintLayout
+import net.skyscanner.backpack.R
+import net.skyscanner.backpack.calendar.presenter.DateFormatter
+import net.skyscanner.backpack.calendar.presenter.HighlightedDaysAdapter
+import net.skyscanner.backpack.text.BpkText
+import net.skyscanner.backpack.util.unsafeLazy
+
+class HighlightedDaysMonthFooter(
```
Comment (bvitaliyg) [inline Backpack/src/main/java/net/skyscanner/backpack/calendar/view/HighlightedDaysMonthFooter.kt]: Let's make it open

File: Backpack/src/main/java/net/skyscanner/backpack/calendar/view/HighlightedDaysMonthFooter.kt [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    orientation = VERTICAL
+    setPadding(spacingBase, 0, spacingBase, spacingLg)
+  }
+
+  private fun update() {
+    removeAllViews()
+    holidays?.forEach { holiday ->
+      addView(
+        HolidayView(context).apply {
+          setDescription(holiday.description)
```
Comment (bvitaliyg) [inline Backpack/src/main/java/net/skyscanner/backpack/calendar/view/HighlightedDaysMonthFooter.kt]: Optional: can use use properties for that?

File: Backpack/src/main/java/net/skyscanner/backpack/calendar/view/HighlightedDaysMonthFooter.kt (hunk 2 above)
Comment (tiagoengel) [reply Backpack/src/main/java/net/skyscanner/backpack/calendar/view/HighlightedDaysMonthFooter.kt]: Good point, done!

**Pull request merged**